backend/search.py

```diff
@@ -1,60 +1,46 @@
-from googleapiclient.discovery import build
-import pprint
-import pickle
-import re
-import numpy
+
 import warnings
 warnings.filterwarnings("ignore")
 import urllib.parse
 from senti import *
 
-
-
-
+# build the YT api using the api key and use the youtube data -> v3 api
 api_key = 'AIzaSyB99vDvUQUO8qWZ7QDb2l5UvO1QEzDAErQ'
 youtube = build('youtube', 'v3', developerKey=api_key)
```

Comment:
Please use api key as env variable

Reply:
Okay I'm on it.

Reply:
@a-ma-n  not resolved yet? Shouldn't take much time

Reply:
Done 👍